templates/pages/student-form.html

```diff
@@ -0,0 +1,118 @@
+				  </div>
+	
+				  <div class="col-span-6 sm:col-span-4 my-2">
+					<label for="email_address" class="block text-sm font-medium text-gray-700">Student Email:</label>
+					<input type="text" name="email_address" id="email_address" autocomplete="email" class="mt-1 p-2 focus:ring-indigo-500 focus:border-indigo-500 block w-full shadow-sm sm:text-sm border-gray-300 rounded-md frminp">
+				  </div>
+
+				  <div class="col-span-6 sm:col-span-3 my-2">
+					<fieldset id="courses">
+						<legend class="block text-sm font-medium text-gray-700">Courses:</legend>
+						<div class="mt-4 space-y-4">
+
+{#first checkbox#}
+						  <div class="flex items-start">
+							<div class="flex items-center h-5">
+							  <input id="CSCI350" name="CSCI350" type="checkbox" class="focus:ring-indigo-500 h-4 w-4 text-indigo-600 border-gray-300 rounded js_subject_checkbox">
+							</div>
+							<div class="ml-3 text-sm">
+							  <label for="CSCI350" class="font-medium text-gray-700">CSCI350</label>
+							</div>
+						  </div>
+						</div>
+
+                        <div class="flex items-start">
+							<div class="flex items-center h-5">
+							  <input id="CSCI350" name="CSCI350" type="checkbox" class="focus:ring-indigo-500 h-4 w-4 text-indigo-600 border-gray-300 rounded js_subject_checkbox">
+							</div>
+							<div class="ml-3 text-sm">
+							  <label for="CSCI350" class="font-medium text-gray-700">CSCI350</label>
+							</div>
+						  </div>
+						  <div class="flex items-start">
+							<div class="flex items-center h-5">
+							  <input id="CSCI400" name="CSCI400" type="checkbox" class="focus:ring-indigo-500 h-4 w-4 text-indigo-600 border-gray-300 rounded js_subject_checkbox">
+							</div>
+							<div class="ml-3 text-sm">
+							  <label for="CSCI400" class="font-medium text-gray-700">CSCI400</label>
+							</div>
+						  </div>
+
+						  <div class="flex items-start">
+							<div class="flex items-center h-5">
+							  <input id="CSCI150" name="CSCI150" type="checkbox" class="focus:ring-indigo-500 h-4 w-4 text-indigo-600 border-gray-300 rounded js_subject_checkbox">
+							</div>
+							<div class="ml-3 text-sm">
+							  <label for="CSCI150" class="font-medium text-gray-700">CSCI150</label>
+							</div>
+						  </div>
+
+
+
+					  </fieldset>
+				</div>
+			</div>
+			<div class="px-4 py-3 bg-gray-50 text-right sm:px-6">
+			  <button type="submit" class="inline-flex justify-center py-2 px-4 border border-transparent shadow-sm text-sm font-medium rounded-md text-white bg-indigo-600 hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500">
+				Submit
+			  </button>
+			</div>
+		  </div>
+		</form>
+	  </div>
+  </div>
+  {% include "components/footer.html" %}
+
+
+  {# making sure no more than 8 are selceted as no one can take more thsn 8 courses  #}
+  <script>
```

Comment:
I think we should keep all our javascript in separate files. This will get out of hand quickly otherwise